spi/w5500_spi/README.adoc

```diff
@@ -0,0 +1,59 @@
+= Attaching a W5500 via SPI
+
+This example code shows how to interface the Raspberry Pi Pico to the WIZ850io(W5500) board. The particular device used can be interfaced via SPI, we are using SPI, and interfacing at 3.3v.
```

Comment:
Needs to mention the alternative board that we've now tested with, and any alternative arrangements needed, e.g. wiring etc?